stl/inc/ratio

```diff
@@ -19,59 +20,54 @@ _STL_DISABLE_CLANG_WARNINGS
 #undef new
 
 _STD_BEGIN
-template <intmax_t _Val>
-struct _Abs : integral_constant<intmax_t, (_Val < 0 ? -_Val : _Val)> {}; // computes absolute value of _Val
-
-template <intmax_t _Ax, intmax_t _Bx, bool _Sfinae = false,
-    bool _Good = (_Abs<_Ax>::value <= INTMAX_MAX / (_Bx == 0 ? 1 : _Abs<_Bx>::value))>
-struct _Safe_mult : integral_constant<intmax_t, _Ax * _Bx> {}; // computes _Ax * _Bx without overflow
+_NODISCARD constexpr intmax_t _Abs(intmax_t _Val) { // computes absolute value sof _Val
+    return (_Val < 0 ? -_Val : _Val);

```

Comment:
Note that this is broken for the most negative value, appreciate a comment why it is fine.
I think this is sorted out by `static_assert` in `ratio`.
Or maybe add a check here as well?

Reply:
No need extra parens